src/OrchardCore/OrchardCore.Localization.Abstractions/OrchardCoreRequestLocalizationOptions.cs

```diff
@@ -20,14 +21,33 @@ public OrchardCoreRequestLocalizationOptions() : this(ignoreSystemSettings: fals
     }
 
     /// <summary>
-    /// Creates a new <see cref="OrchardCoreRequestLocalizationOptions"/> with default values and ability to ignore system settings.
+    /// Creates a new <see cref="OrchardCoreRequestLocalizationOptions"/> with default values and the ability to ignore system settings.
     /// <param name="ignoreSystemSettings">Whether to ignore the system culture settings or not.</param>
     /// </summary>
     public OrchardCoreRequestLocalizationOptions(bool ignoreSystemSettings) : base()
     {
         _useUserOverride = !ignoreSystemSettings;
     }
 
+    /// <summary>
+    /// Initializes the options properties from a provided <see cref="RequestLocalizationOptions"/> instance.
+    /// <param name="requestLocalizationOptions">The provided <see cref="RequestLocalizationOptions"/>.</param>
+    /// </summary>
+    public OrchardCoreRequestLocalizationOptions WithRequestLocalizationOptions(RequestLocalizationOptions requestLocalizationOptions)
```

Comment:
@jtkech I'm not sure while this move here, if we do so, we could passed on the constructor, what do you think?

Reply:
> we could passed on the constructor

Yes I did it at some point but didn't want to remove your `WithRequestLocalizationOptions()` pattern ;)

So as you want.

Reply:
PR is created to address this